modules/aql/arrow-query-language/src/test/kotlin/arrow/aql/tests/IOExample.kt

```diff
@@ -39,7 +39,7 @@ class RemoteDataSource : DataSource {
     mapOf(User(UserId("user2")) to listOf(Task("Remote Task assigned to user2")))
 
   override fun allTasksByUser(user: User): IO<List<Task>> =
-    IO.async { cb ->
+    IO.async { _, cb ->
```

Comment:
I think this one should be ignored. As in: no matter what connection the new async gets passed, it'll reuse the one passed in for the current `loop()` call.